pkg/controllers/deprovisioning/consolidation.go

```diff
@@ -220,7 +224,7 @@ func (c *Consolidation) ValidateCommand(ctx context.Context, nodesToDelete []Can
 	// a 4xlarge and replace it with a 2xlarge. If things have changed and the scheduling simulation we just performed
 	// now says that we need to launch a 4xlarge. It's still launching the correct number of nodes, but it's just
 	// as expensive or possibly more so we shouldn't validate.
-	if !instanceTypesAreSubset(cmd.replacementNode.InstanceTypeOptions, newNodes[0].InstanceTypeOptions) {
+	if !instanceTypesAreSubset(cmd.replacementNodes[0].InstanceTypeOptions, newNodes[0].InstanceTypeOptions) {
```

Comment:
It's not clear why we're only getting the first node here.

Reply:
discussed offline. We're guaranteed its 1. I think we can potentially collapse all of these cases with a for loop.